static/templates/draft_table_body.hbs

```diff
@@ -11,7 +11,7 @@
                     <br />
                     {{#tr}}Drafts older than <strong>{draft_lifetime}</strong> days are automatically removed.{{/tr}}
                     <br />
-                    {{t "Pro tip: You can use 'd' to open your drafts."}}
+                    {{t "You can also use 'd' to open your drafts."}}
```

Comment:
I remember folks talking about changing this some time ago but couldn't find the conversation. Thoughts?

Reply:
I think the idea was that "pro tip" has a connotation that pros use keyboard shortcuts.

Reply:
Hm, I'm wondering if it's worth having this line there at all. It's a lot of space/attention to dedicate to the shortcut, and it seems more appropriate to just advertise it in tooltips. What do you think?

Reply:
Filed #22131 for the tooltip change, which we should do regardless.

Reply:
I like the idea of moving it to the tooltip and removing it from here completely!

Reply:
Cool, let's go ahead and remove it, then. You can put it in a separate commit in case @timabbott has a different perspective.